orm/codec.proto

```diff
@@ -11,3 +13,12 @@ message MultiRef {
 message Counter {
   int64 count = 1;
 }
+
+// VersionedID is the combination of document ID and version number.
+message VersionedIDRef {
```

Comment:
Would it make sense to rename it to `VersionedID` and use if both as an object ID and a reference type, the same as single value ID is used?
A versioned bucket could then require a `GetVersionedID() VersionedID` method to be present, which is is generated by the protobuf compiler.